app/Helper/TaskHelper.php

```diff
@@ -238,6 +238,7 @@ public function renderDueDateField(array $values, array $errors = array(), array
     public function renderPriority($priority)
     {
         $html = '<span class="task-priority" title="'.t('Task priority').'">';
+        $html .= '<span class="ui-helper-hidden-accessible">'.t('Task priority').': </span>';
```

Comment:
The colon should be part of the translated text because typographic rules are different between languages.

For example, in French, there is a non-breaking space before the colon: `Priorité de la tâche : P1`.

Reply:
Thanks for the info :slightly_smiling_face: I never knew that differed between languages.

I've removed the colons so we aren't introducing further translation terms and can simply make use of the existing title terms.